### PR TITLE
tests: ERSPAN Type I packet parsing

### PR DESCRIPTION
**Pull request by jlucovsky**
This PR adds tests cases related to ERSPAN Type I packet parsing.

Requires suricata PR [4466](https://github.com/OISF/suricata/pull/4466)

### CODE REVIEW

**Comment by jlucovsky** [comment]
The Suricata PR is https://github.com/OISF/suricata/pull/4475

File: tests/detect-erspan-typeI-01/test.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+requires:
+
+  min-version: 5.0.2
```
Comment (victorjulien) [inline tests/detect-erspan-typeI-01/test.yaml]: Can you update this to 6.0.0? It's been merged into master.

File: tests/detect-erspan-typeI-01/test.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+
+checks:
+
+    - filter:
+        count: 2
+        match:
+            event_type: flow
```
Comment (victorjulien) [inline tests/detect-erspan-typeI-01/test.yaml]: Add in ip checks too?

**Comment by jlucovsky** [comment]
Continued in #177

**Pull request closed without merging**